### PR TITLE
list3

### PR DESCRIPTION
**Pull request by simalyolgosteren**

### CODE REVIEW

File: editor-components/list/list3/list3.tsx [hunk 1]
```diff
@@ -132,42 +148,92 @@ class List3 extends BaseList {
...
+                "itemCount"
+              )}, 1fr)`,
+            }}
+          >
+            {(title || description) && (
+              <div className={this.decorateCSS("first")}>
+                <div className={this.decorateCSS("firstContent")}>
+                  {title && (
+                    <div className={this.decorateCSS("title")}>
+                      <h1>{title}</h1>
```
Comment (dogusmiuw) [inline editor-components/list/list3/list3.tsx:177]: should use this.getPropValue("title")

File: editor-components/list/list3/list3.tsx [hunk 2]
```diff
@@ -132,42 +148,92 @@ class List3 extends BaseList {
...
+                    </div>
+                  )}
+                  {description && (
+                    <div className={this.decorateCSS("description")}>
+                      {description}
```
Comment (dogusmiuw) [inline editor-components/list/list3/list3.tsx:182]: should use this.getPropValue("description")

File: editor-components/list/list3/list3.tsx [hunk 3]
```diff
@@ -132,42 +148,92 @@ class List3 extends BaseList {
...
+                    </div>
+                  )}
+                  {this.castToString(this.getPropValue("button")) && (
```
Comment (dogusmiuw) [inline editor-components/list/list3/list3.tsx:185]: reference casting operation between render and return statements

**Review by dogusmiuw — CHANGES_REQUESTED** [review]

File: editor-components/list/list3/list3.tsx [hunk 4]
```diff
@@ -132,42 +147,98 @@ class List3 extends BaseList {
...
+                "itemCount"
+              )}, 1fr)`,
+            }}
+          >
+            {(title || description) && (
+              <div className={this.decorateCSS("first")}>
+                <div className={this.decorateCSS("firstContent")}>
+                  {title && (
+                    <div className={this.decorateCSS("title")}>
+                      <h1>{this.getPropValue("title")}</h1>
```
Comment (Ayazelar) [inline editor-components/list/list3/list3.tsx:179]: should have className with decorateCss method

File: editor-components/list/list3/list3.tsx [hunk 5]
```diff
@@ -132,42 +147,98 @@ class List3 extends BaseList {
...
+                    </div>
+                  )}
+                  {description && (
+                    <div className={this.decorateCSS("description")}>
+                      <p>{this.getPropValue("description")}</p>
```
Comment (Ayazelar) [inline editor-components/list/list3/list3.tsx:184]: should have className with decorateCss method

**Review by Ayazelar — CHANGES_REQUESTED** [review]
.

**Review by Ayazelar — APPROVED** [review]

**Pull request merged**